### PR TITLE
OMID-280 Use Hbase 2.5 for building OMID

### PR DESCRIPTION
**Pull request by stoty**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -174,8 +174,7 @@
         <hamcrest.version>1.3</hamcrest.version>
         <guava.version>32.1.3-jre</guava.version>
         <!-- 2.12+ shades guava -->
-        <curator.version>4.2.0</curator.version>
-        <zookeeper.version>3.5.9</zookeeper.version>
```
Comment (stoty) [inline pom.xml:178]: This was not used.

**Review by virajjasani — APPROVED** [review]
+1, it also works with Phoenix 5.2.0 + HBase 2.4 profile?

**Comment by stoty** [comment]
> +1, it also works with Phoenix 5.2.0 + HBase 2.4 profile?

Yes, it doesn't matter.
We dependecy manage every Omid transitive dependency in Phoenix.

The only thing that this really affects is the libraries in the TSO server assembly and the test environment.

Just as when we build Phoenix for HBase 2.5 but Omid is built for 2.4.
(Or when we build Phoenix with HBase 2.1 in 5.1, but Omid is built with 2.4.)

**Comment by stoty** [comment]
I have merged this a long time ago, but forgot to close this PR.

**Pull request closed without merging**